src/VisualStudio/Core/Def/Implementation/InlineRename/InlineRenameUndoManager.cs

```diff
@@ -218,27 +218,26 @@ private IOleUndoManager GetUndoManager(ITextBuffer subjectBuffer)
 
             private IEnumerable<IOleUndoUnit> GetUndoUnits(IOleUndoManager undoManager)
             {
-
                 // Unfortunately, EnumUndoable returns the units in oldest-first order.
-                undoManager.EnumUndoable(out var undoUnitEnumerator);
-                if (undoUnitEnumerator == null)
-                {
-                    yield break;
-                }
+                IEnumOleUndoUnits undoUnitEnumerator = null;
+                var hr = ErrorHandler.CallWithCOMConvention(() => undoManager.EnumUndoable(out undoUnitEnumerator));
```

Comment:
💡 This could also be implemented as:

```csharp
IEnumOleUndoUnits undoUnitEnumerator;
try
{
  undoManager.EnumUndoable(out undoUnitEnumerator);
}
catch (COMException)
{
  yield break;
}